cfme/tests/automate/test_service_dialog.py

```diff
@@ -481,3 +482,57 @@ def test_automate_methods_from_dynamic_dialog_should_run_as_per_designed(
         for ele_name in [ele_label, "label2", "label3"]:
             # Checking if automate method is not triggered after updating values of dialog widgets
             view.fields(ele_name).input.fill(fauxfactory.gen_alphanumeric())
+
+
+@pytest.mark.meta(automates=[BZ(1677724)])
+@pytest.mark.customer_scenario
```

Comment:
You can directly write like this:
```
@pytest.mark.meta(automates=[1677724])
```
- So no need to **import BZ** only for this.